tests/testthat/test-to_md.R

```diff
@@ -68,3 +68,42 @@ test_that("to_md does not break tables", {
   to_md(yaml_xml_list, newmd)
   testthat::expect_snapshot_file(newmd)
 })
+
+test_that("code chunks can be inserted on round trip", {
+
+  path <- system.file("extdata", "example2.Rmd", package = "tinkr")
+  tmpdir <- tempdir()
+  suppressWarnings(dir.create(tmpdir))
+  newmd <- file.path(tmpdir, "ex.Rmd")
+  on.exit(file.remove(newmd))
```

Comment:
should we use self-cleaning test fixtures? https://www.tidyverse.org/blog/2020/04/self-cleaning-test-fixtures/

Reply:
in particular https://withr.r-lib.org/reference/with_file.html

Reply:
+10000 
|
I'm definitely for the self-cleaning test fixtures, but I would prefer to convert them in a separate PR after this is merged.

Reply:
I opened #25 to track this